### ISSUE TITLE
Refactor meson scripts on Windows

### ISSUE THREAD

**Issue by xarkes** [radare/radare2#8690]
The idea is to remove `meson.bat` file and create a new `meson.py` file which will merge `sys\meson*` and `meson.bat` into a single and unique file. 
This is required because `meson.bat` works in most general cases, batch is not easy to maintain, etc. And meson works with python so it sounds logical to do everything in python.
- [ ] Clean meson Makefile rules
- [x] Translate `meson.bat` into python
- [x] Translate `sys\meson_install.bat` into python
- [x] Merge all the `meson*.py` files into one
And maybe:
- [x] Remove plugins.meson.cfg file
- [x] Merge all of this with the `meson.py` file which is distributed with `mesonbuild` (Will avoid Windows users to manually copy `meson.py` file in radare2 root directory before starting the script. Plus it sounds stupid to do a `os.system('python meson.py')`  when you use python already.)

**Comment by xarkes** [comment]
cc @mrexodia What we've been talking about yesterday

**Comment by pelijah** [comment]
Prepared experimental Python variant of meson_install today. How it looks:
```Python
os.makedirs(DIST)
_move(r'{BIN}\*.exe', r'{DIST}')
_move(r'{BIN}\*.dll', r'{DIST}')
if copylib:
    _move(r'{LIB}\*.lib', r'{DIST}')
    _move(r'{LIB}\*.a', r'{DIST}')
_copytree(r'{ROOT}\shlr\www', r'{DIST}\www')
_copytree(r'{ROOT}\libr\magic\d\default', r'{DIST}\share\radare2\{R2_VERSION}\magic')
_copy(r'{ROOT}\libr\syscall\d\*.sdb', r'{DIST}\share\radare2\{R2_VERSION}\syscall')
_copy(r'{ROOT}\libr\anal\d\*.sdb', r'{DIST}\share\radare2\{R2_VERSION}\fcnsign')
_copy(r'{ROOT}\libr\anal\d\*.sdb', r'{DIST}\share\radare2\{R2_VERSION}\opcodes')
```
Seems to be working. _copy(), _move(), _copytree() are just wrappers around functions from shutil module.

**Comment by pelijah** [comment]
#9183 was merged. Do you have new ideas?

**Comment by xarkes** [comment]
I think this can be closed?
I saw meson_options.txt on the root directory, imho it should be deleted.

**Comment by pelijah** [comment]
Added commit with fix to my PR #9258

**Comment by xarkes** [comment]
Ok I merged, I'm closing this.
